src/mysql/mod.rs

```diff
@@ -0,0 +1,120 @@
+use std::{collections::HashMap, time::Duration};
+
+use testcontainers::{core::WaitFor, Image};
+
+const NAME: &str = "mysql";
+const TAG: &str = "8.1";
+
+#[derive(Debug)]
+pub struct Mysql {
+    env_vars: HashMap<String, String>,
+}
```

Comment:
Could you add some basic documentation? We've not added documentation for all our images yet, but it's planned & would be nice to have for new images.

For example, take a look at this one:
https://github.com/testcontainers/testcontainers-rs-modules-community/blob/300f744ee649465e085668095fbefbabd146aebc/src/rabbitmq/mod.rs#L6-L31

I found useful to add at least references.

Reply:
added docs